template/lambda-sqs-worker-cdk/package.json

```diff
@@ -6,7 +6,7 @@
     "deploy": "cdk deploy appStack --require-approval never --context stage=${ENVIRONMENT}",
     "format": "skuba format",
     "lint": "skuba lint",
-    "package": "yarn install --ignore-optional --ignore-scripts --modules-folder ./lib/node_modules --non-interactive --offline --production",
+    "package": "pnpm install --ignore-scripts --modules-folder ./lib/node_modules --no-optional --offline --production",
```

Comment:
This package script seems dicey; we'll have to manually validate that we can implement an equivalent with pnpm